### DISCUSSION TITLE
Backup while pods are running

### DISCUSSION THREAD

**Discussion by 2020testuser** [vmware-tanzu/velero#3487] [Community support Q&A] [answered]
Hi,
Could anyone shed some light on the below?

How the Velero namespace backup will be performed when the pods are running and using the database connections. Will the database connections and other connections be temporarily paused to perform the backup?

Would like to understand how the backup would be done when jobs / pods are running.
Thanks in advance!

**Comment by ashish-amarnath** [comment, marked as answer]
Velero will not interrupt or pause your application pods while backing them up unless you specifically ask it to.
Using [velero backup hooks](https://velero.io/docs/v1.5/backup-hooks/) it is possible to quiesce and un-quiesce databases hosted by pods Velero is backing up.
Similarly, using [velero restore hooks](https://velero.io/docs/v1.5/restore-hooks/) you can ensure that the data is ready before the database application containers are restored.
